static/templates/single_message.handlebars

```diff
@@ -22,7 +22,7 @@
                                         {{{ status_message }}}
                                     </span>
                                     {{#if_and last_edit_timestr include_sender}}
-                                    <div class="message_edit_notice" title="{{#tr this}}Edited (__last_edit_timestr__){{/tr}}">{{t "EDITED" }}</div>
+                                    <div class="message_edit_notice" aria-label="{{#tr this}}Edited (__last_edit_timestr__){{/tr}}">{{t "EDITED" }}</div>
```

Comment:
We need the title attribute here for displaying the data on when this was last edited on mouseover.  So we shouldn't be removing the `title` attributes here.

Reply:
(I think this applies to most of the below changes).

Reply:
According to @alexjurgensen on #9222, he suggested that I replace all `title` attributes with `aria-label` attributes. I also consulted various other accessibility guides, which also suggested that `title` attributes were detrimental to the experience of screenreaders (examples: see [here](https://developer.paciellogroup.com/blog/2012/01/html5-accessibility-chops-title-attribute-use-and-abuse/), [here](https://a11yproject.com/posts/title-attributes/), and [here](https://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/))

My research also suggested the better approach would be to use tooltips instead of `title` tags on those elements to reveal information to sight-seeing users instead. Thoughts?

Reply:
I agree `title` is unfortunately not ideal for accessibility; just trying to make clear that this change would break important functionality for sight-seeing users.

Replacing the title attributes in message bodies with tooltip + aria-label would be a reasonable solution.

Reply:
Update on this PR: Bootstrap's native tooltips use the `title` attribute to specify tooltip content, which is unwanted. I'm not sure which would be preferred in this case: manually creating JS tooltips for all message icons is or making an entirely new CSS/JS classes for tooltips. What do you suggest>